test/distributed/org/apache/cassandra/distributed/test/trackwarnings/AbstractClientSizeWarning.java

```diff
@@ -283,15 +295,26 @@ public boolean matches(InetAddress value)
         }
         assertHistogramUpdated();
         assertWarnAborts(0, 2, 1);
+    }
+
+    public void failThresholdDisabled(String cql) throws UnknownHostException
+    {
+        ICoordinator node = CLUSTER.coordinator(1);
+        for (int i = 0; i < failThresholdRowCount(); i++)
+            node.execute("INSERT INTO " + KEYSPACE + ".tbl (pk, ck, v) VALUES (1, ?, ?)", ConsistencyLevel.ALL, i + 1, bytes(512));
+
+        if (shouldFlush())
+            CLUSTER.stream().forEach(i -> i.flush(KEYSPACE));
 
         // query should no longer fail
         enable(false);
+        checkpointHistogram();
```

Comment:
this has to go after the query

Reply:
I was thinking that it is maybe better to remove this checkpointHistogram completely from here, and just not change what current test is doing
https://github.com/apache/cassandra/blob/trunk/test/distributed/org/apache/cassandra/distributed/test/trackwarnings/AbstractClientSizeWarning.java#L195